.github/workflows/test.yaml

```diff
@@ -68,6 +72,21 @@ jobs:
         cd example
         bazel build //:example.wasm
 
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GHCR_TOKEN }}
```

Comment:
looks like the `secrets.GHCR_TOKEN` is not configured even in the branches in this repository. Can you check if it is enabled? @bianpengyuan 

![img](https://user-images.githubusercontent.com/13513977/138195063-365ba0d5-25ab-4df7-93c4-a5fbf7d7957d.png)

Reply:
Oh I thought it will be automatically configured, but maybe someone has to set manually :D

Reply:
I added the secret and looks like login succeeded.